### DISCUSSION TITLE
npx tailwindcss init throw an error, Cannot find module './util'

### DISCUSSION THREAD

**Discussion by kraken-afk** [tailwindlabs/tailwindcss#11150] [Help]
im using vite with React + TypeScript, and also installing tailwind-css with
```sh
npm i -D tailwindcss postcss autoprefixer
```
then i ran
```sh
npx tailwindcss init
```
however, it didn't work as i expected
```sh
node:internal/modules/cjs/loader:998
  throw err;
  ^

Error: Cannot find module './util'
Require stack:
---
code: 'MODULE_NOT_FOUND',

Node.js v18.12.1
```
im using npm create vite@latest to scaffolding the project, here's my package-json, also i didn't forget to ran thenpm install
```json
{
  "dependencies": {
    "react": "^18.2.0",
    "react-dom": "^18.2.0"
  },
  "devDependencies": {
    "@types/react": "^18.0.28",
    "@types/react-dom": "^18.0.11",
    "@typescript-eslint/eslint-plugin": "^5.57.1",
    "@typescript-eslint/parser": "^5.57.1",
    "@vitejs/plugin-react": "^4.0.0",
    "autoprefixer": "^10.4.14",
    "eslint": "^8.38.0",
    "eslint-plugin-react-hooks": "^4.6.0",
    "eslint-plugin-react-refresh": "^0.3.4",
    "postcss": "^8.4.23",
    "tailwindcss": "^3.3.2",
    "typescript": "^5.0.2",
    "vite": "^4.3.2"
  }
}
```

**Comment by kraken-afk** [comment]
i decide to import directly the tailwindcss to my tsx file, still doesn't work
![Screenshot (190)](https://user-images.githubusercontent.com/77183252/235942808-cffc889a-9926-4cfd-85ec-1c5f9bbc171a.png)

**Comment by wongjn** [comment]
Perhaps check if you have an `import './util` line somewhere that is not resolving to a `util.js` or `util.ts` file. Otherwise, consider providing a remotely-hosted git repository that reproduces the unexpected behavior.

**Reply by kraken-afk** [reply]
No sir, i didn't have it. Here's the [repo](https://github.com/kraken-afk/porto).

**Reply by kraken-afk** [reply]
i assume the error comes from somewhere in node_modules

**Reply by wongjn** [reply]
Unfortunately I was unable to reproduce your issue, but I hope you find a resolution.
